probe-rs/src/probe/jlink/mod.rs

```diff
@@ -524,7 +528,7 @@ impl DebugProbe for JLink {
                 // We don't actually care about the response here.
                 // A read on the DPIDR will finalize the init procedure and tell us if it worked.
                 jlink.swd_io(direction, swd_io_sequence)?;
-                log::debug!("Sucessfully swapped to SWD.");
+                log::debug!("Sucessfully switched to SWD");
```

Comment:
```suggestion
                log::debug!("Sucessfully switched to SWD.");
```

I like complete English sentences (yes I know the subject is missing ...) :)

Reply:
It looks quite weird in the logs though, because it's being used quite inconsistently. So far almost all projects I've seen that log things don't use punctuation for single sentences (and practically all logs are a single sentence). It adds visual noise that doesn't add anything useful.

If you insist I'll happily re-add the `.` though :slightly_smiling_face: But then it would be good to consistently use punctuation signs for all logs in probe-rs (and associated projects).

Reply:
I agree that this is not used consistently in projects and I always have to fight for it to be correct in probe-rs too.

I guess this is just my OCD that is instantly triggered :/ Guess we can leave it like this :)